docs/content/docs/blog/release-osp-1.5.md

```diff
@@ -0,0 +1,96 @@
+---
+title: OpenShift Pipeline 1.5 PAC release
+---
+
+OpenShift Pipeline 1.5 introduces many new features for Pipelines-as-Code, here
+is a list of the most important ones.
+
+## GitOPS Commands
+
+The GitOPS commands are those small comments you can do a on pull request to
+trigger some actions on your pipelinerun via Pipelins-as-Code.
```

Comment:
What actions do you mean other than triggering a pipelinerun? If this only means triggering a pipelinerun, then just make it "trigger a pipelinerun".